deploy/olm-catalog/kfctl/0.1.0/kfctl.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: "Machine Learning"
+    description: "Kubeflow Operator for deployment and management of Kubeflow"
+    support: Kubeflow
+  name: kfctl.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
```

Comment:
`owned` field is missing from `customresourcedefinitions`. You can refer to the [operator example](https://github.com/operator-framework/operator-sdk-samples/blob/master/go/memcached-operator/deploy/olm-catalog/memcached-operator/0.12.0/memcached-operator.v0.12.0.clusterserviceversion.yaml#L24-L59) to fill in the necessary fields. 

It looks like some of these generated code is missing during `operator-sdk generate csv`.  The [command](https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/olm-catalog/csv_updaters.go#L245-L248) is looking for the go apis spec under the defined [ApisDir](https://github.com/operator-framework/operator-sdk/blob/master/internal/scaffold/constants.go#L29) which is a constant looking the apis under /pkg/apis/kfdef where the actual kfdef's apis in kfctl is /pkg/apis/app/kfdef. 

Probably we should open an issue on Operator-SDK to suggest some of these variables to be configurable for operators that are generated on an existing package.

Reply:
Thanks Tommy for the pointer to the OLM bug, and potential fix.- this should be updated now